README.md

```diff
@@ -110,11 +110,8 @@ Use the following to configure which queue async jobs are added to
 
 Use the following configuration options to choose the [operation modes](#operation-modes):
 
-	ckanext.validation.run_on_create_async = True|False (Defaults to True)
-	ckanext.validation.run_on_update_async = True|False (Defaults to True)
-
-	ckanext.validation.run_on_create_sync = True|False (Defaults to False)
-	ckanext.validation.run_on_update_sync = True|False (Defaults to False)
+    ckanext.validation.default_create_mode = async|sync (Defaults to "sync")
```

Comment:
This is not backwards compatible. Please re add run_on_create_async and  run_on_update_async settings (bool) and convert to default_create_mode, default_update_mode word type.

Reply:
Readded `run_on_update_async` and `run_on_create_async` options.